osf/management/commands/sync_citation_styles.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+from django.core.management.base import BaseCommand
+import re
+import os
+import io
+import zipfile
+import requests
+import logging
+from django.apps import apps
+from api.base import settings
+
+from django.db import transaction
+from lxml import etree
+from urllib.parse import urlparse
```

Comment:
Minor: do we still enforce import orders in osf.io?

Reply:
I don't believe we ever did that.

Reply:
I agree for existing files, we don't want to re-order them to avoid creating conflicts for others. For newly created files such as this, I suggest we start ordering imports. The first block is built-in, the second is third-party, the third is our own libraries.

```python
import io
import logging
import os
import re
import requests
import zipfile

from django.apps import apps
from django.core.management.base import BaseCommand
from django.db import transaction
from lxml import etree
from urllib.parse import urlparse

from api.base import settings
```